ssl/src/main/java/io/crate/protocols/postgres/SslReqHandlerImpl.java

```diff
@@ -21,22 +21,31 @@
 
 package io.crate.protocols.postgres;
 
+import io.crate.protocols.ssl.SslConfigSettings;
 import io.netty.buffer.ByteBuf;
 import io.netty.channel.ChannelPipeline;
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslHandler;
 import org.apache.logging.log4j.Logger;
 import org.elasticsearch.common.logging.Loggers;
 import org.elasticsearch.common.settings.Settings;
 
 /**
- * Handler that processes an optional SSLRequest and rejects SSL.
+ * Handler that processes an optional SSLRequest and configures SSL.
  */
-public final class SslReqRejectingHandler implements SslReqHandler {
+public final class SslReqHandlerImpl implements SslReqHandler {
```

Comment:
Generally, it is nice to have an interface, especially for testing. Since we don't need it for the tests, I'm ok with removing it.